src/backend/core/migrations/0010_team_depth_team_numchild_team_path_and_more.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 5.1.3 on 2024-11-26 13:55
+
+from django.db import migrations, models
+
+from treebeard.mp_tree import MP_Node
+
+
+def update_team_paths(apps, schema_editor):
+    Team = apps.get_model("core", "Team")
+    alphabet = MP_Node.alphabet
+    steplen = MP_Node.steplen
+
+    # for default values, this resolves to: "{:04d}"
+    path_format = f"{{:{alphabet[0]}{steplen}d}}"
```

Comment:
I'm not sure this works because the alphabet contains letters :thinking: 
the path could be "1aZ6"